NVActivityIndicatorView/NVActivityIndicatorView.swift

```diff
@@ -75,54 +138,17 @@ public class NVActivityIndicatorView: UIView {
         Create a activity indicator view with specified type, color, size and size
         
         :param: frame view's frame
-        :param: type animation type, value of NVActivityIndicatorType enum
-        :param: color color of activity indicator view
-        :param: size actual size of animation in view
+        :param: type animation type, value of NVActivityIndicatorType enum. Default type is pacman.
+        :param: color color of activity indicator view. Default color is white.
+        :param: size actual size of animation in view. Default size is 40
     
         :returns: The activity indicator view
     */
-    public init(frame: CGRect, type: NVActivityIndicatorType, color: UIColor?, size: CGSize?) {
+    public init(frame: CGRect, type: NVActivityIndicatorType = DEFAULT_TYPE, color: UIColor = DEFAULT_COLOR, size: CGSize = DEFAULT_SIZE) {
```

Comment:
Yes, this will fail to compile. But the beauty about using the default values is that you no longer need to pass explicit nil and the parameter can be omitted altogether. Instead of doing this:

```
NVActivityIndicatorView(frame: <value>, type: <value>, color: nil, size: <value>)
```

you can do this:
```
NVActivityIndicatorView(frame: <value>, type: <value>,  size: <value>)
```

or if you want you can also skip the size:

```
NVActivityIndicatorView(frame: <value>, type: <value>)
```

I believe this is a bit cleaner than passing explicit nils. The caller can just skip the parameter he doesn't have a value for.